app/views/posts/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Posts#index</h1>
-<p>Find me in app/views/posts/index.html.erb</p>
+<h1>Posts</h1>
+
+<% @posts.each do |post| %>
+    <h2><%= post.title %></h2>
+    <p><%= post.text %></p>
+    <p><%= link_to 'Read more...', post_path(post) %></p>
+    <p><%= post.comments_counter %> Comment
+    <%= post.likes_counter %> Like 
+    </p>
+    <p>By: <%= post.user.name %> </p>
+<% end %>
```

Comment:
- [ ] [REQUIRED] Great implementation, However, you need to ensure that Clicking on each user takes us to the show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png), **See all Post** links open user's post index as displayed on the wireframe as well, Post-show page displays post with all comments listed as shown on the wireframe. Let's ensure that the design matches the wireframe design provided as this is part of the requirement.

### Your Design
![Screenshot from 2022-09-23 17-24-01](https://user-images.githubusercontent.com/58692590/192009460-ed5d49b2-24ef-440d-9ae0-313b6bd11bb1.png)

### Wireframe Design
![Screenshot from 2022-09-23 17-26-04](https://user-images.githubusercontent.com/58692590/192009552-9570bedb-9e15-4716-87c7-6c112e9e2e71.png)

-